README.md

```diff
@@ -48,6 +48,28 @@ Then follow the instructions displayed by the virtual machine.
 
 Sass files are currently generated on each pageload, just like any other template. You do not need to manually compile the Sass.
 
+## Run without Vagrant
+
+You can also run it locally without using Vagrant. First [install Foreman globally](https://github.com/ddollar/foreman#installation):
+
+```bash
+$ gem install foreman
+```
+
+Then install the project's gems:
+
+```bash
+$ bundle install
```

Comment:
I think you'll need to add a couple of commands for cloning the repository before you will be able to start installing gems :)